cosmwasm/packages/wasmi-runtime/src/crypto/aes_siv.rs

```diff
@@ -37,46 +37,28 @@ fn aes_siv_encrypt(
     ad: Option<&[&[u8]]>,
     key: &SymmetricKey,
 ) -> Result<Vec<u8>, CryptoError> {
-    let mut cipher = Aes128Siv::new(GenericArray::clone_from_slice(key));
+    let ad = ad.unwrap_or(&[&[]]);
 
-    match ad {
-        Some(r) => cipher.encrypt(r, plaintext).map_err(|e| {
-            error!("aes_siv_encrypt error: {:?}", e);
-            CryptoError::EncryptionError
-        }),
-        None => cipher.encrypt(&vec![[]], plaintext).map_err(|e| {
-            error!("aes_siv_encrypt error: {:?}", e);
-            CryptoError::EncryptionError
-        }),
-    }
+    let mut cipher = Aes128Siv::new(GenericArray::clone_from_slice(key));
+    cipher.encrypt(ad, plaintext).map_err(|e| {
+        error!("aes_siv_encrypt error: {:?}", e);
+        CryptoError::EncryptionError
+    })
```

Comment:
While I don't disagree with this change, try to avoid changing things that aren't directly related to your issue

Reply:
:+1: